starter-core/src/main/java/io/micronaut/starter/feature/aws/AmazonApiGateway.java

```diff
@@ -18,12 +18,13 @@
 import io.micronaut.core.annotation.NonNull;
 import io.micronaut.starter.application.ApplicationType;
 import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.function.awslambda.AwsLambda;
 import jakarta.inject.Singleton;
 
 @Singleton
-public class AmazonApiGateway extends CdkFeature {
-    public AmazonApiGateway(Cdk cdk) {
-        super(cdk);
+public class AmazonApiGateway extends AwsLambdaRelatedFeature implements AwsApiFeature {
```

Comment:
Is the description here still correct?  It mentions the CDK, but the CDK has been removed from here?

Reply:
You can use it in combination with CDK but you can use it without it.